test/test_ssa_direct_qmc.cc

```diff
@@ -28,14 +29,33 @@ namespace KHineq {
     constexpr double a_b  = (base%2)?((base-1.)/4.):((base*base)/(4.*(base+1.)));
     constexpr double f_b  = a_b/std::log(base);
     constexpr double c_b  = (2.0>1.0+1.0/base+a_b)?(2.0):(1.0+1.0/base+a_b);
+    
+    template <typename Fun>
+    void kh_test(rdmini::ssa_direct<size_t,double> ssa, const Fun& f, double V_f, double exact_mu, std::size_t n_events) {
```

Comment:
The `ssa` object is potentially quite large; better to pass by (const) reference.